conftest.py

```diff
@@ -16,11 +16,11 @@
     "verbosity": Verbosity.normal,
 }
 settings.register_profile("ci_large", max_examples=400, **_common_settings)
-settings.register_profile("ci_pr", max_examples=80, **_common_settings)
-settings.register_profile("ci", max_examples=25, **_common_settings)
-settings.register_profile("dev", max_examples=10, **_common_settings)
+settings.register_profile("ci_pr", derandomize=True, max_examples=80, **_common_settings)
```

Comment:
I don't mind the change coming with this pr, but is this related to dune-gdt?

Reply:
Yes. This stops random errors in CI.